src/large_pages/node_large_page.cc

```diff
@@ -282,20 +294,42 @@ bool IsSuperPagesEnabled() {
 }
 #endif
 
+// Functions in this class must always be inlined because they must end up in
+// the `lpstub` section rather than the `.text` section.
+class MemoryMapPointer {
+ public:
+  FORCE_INLINE MemoryMapPointer() {}
+  FORCE_INLINE MemoryMapPointer(void* start,
+                                size_t size,
+                                int prot,
+                                int flags,
+                                int fd = -1,
+                                int offset = 0) {
+    mem_ = mmap(start, size, prot, flags, fd, offset);
+    size_ = size;
+  }
+  FORCE_INLINE bool operator==(void* rhs) const { return mem_ == rhs; }
+  FORCE_INLINE operator void*() const { return mem_; }
```

Comment:
I have a strong dislike of implicit casts. Can you replace this with e.g.:
```suggestion
  void* mem() const { return mem_; }
```

Reply:
as I understand it, the `FORCE_INLINE` would still be required.

Reply:
Done, with `FORCE_INLINE`.